db/db_log_iter_test.cc

```diff
@@ -330,6 +330,26 @@ TEST_F(DBTestXactLogIterator, TransactionLogIteratorBlobs) {
       "Delete(0, key2)",
       handler.seen);
 }
+
+TEST_F(DBTestXactLogIterator, TransactionIteratorCache) {
```

Comment:
This test would pass before this change too. Maybe check that cache is effective by check some statistics on IO.

Reply:
update it, thx for review